### ISSUE TITLE
Ready event

### ISSUE THREAD

**Issue by HenglyEver** [dansmaculotte/vue-zendesk#18]
Hey, I found this library very useful but I have a little bit of problem with it.

**Comment by gaelreyrol** [comment]
Hi @HenglyEver,

Well, I don't have this kind of problem.

Do you use the config option `hideOnLoad` to automatically hide the widget ?
Did you take a look at the network and performance with dev tools to see if it is not coming from the widget himself ?

**Comment by HenglyEver** [comment]
I don't think it has something to do with `hideOnLoad` or network and performance issue as there was nothing going on when I profiled it. The problem occurs the very first time on every restart, after `loaded` fired, I tried to click on the button that trigger `this.$zendesk.show();` and it took 3-5 seconds to show it, after that `this.$zendesk.hide();` and `this.$zendesk.show();` has no delay and are working fine. Do you know what is the problem or any suggestion to remedy this situation?

**Comment by rtouze** [comment]
Hi,
Like @gaelreyrol , I believe it's a network / widget issue rather than something directly related with this module. When I open the widget for the first time, it triggers 2 GET requests to XXX.zendesk.com:

* one toward `embeddable_blip`
* one toward `tickets_field`

Request toward `embeddable_blip` is triggered everytime you open the widget. `ticket_fields` only the first time. So can you ensure that you do not have latency while retrieving `ticket_fields`?

**Comment by rtouze** [comment]
Hi, I wonder if you have `options.disabled = true` somewhere to prevent getting the widget on Vue context initialization. In this case, the widget is loaded on demand, which can last a couple of seconds.
